modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbMetaStorage.java

```diff
@@ -103,13 +112,85 @@ void putPartitionId(int partitionId) {
         }
     }
 
+    /**
+     * Puts last row ID for which the index was built, {@code null} means index building is finished.
+     *
+     * @param partitionId Partition ID.
+     * @param indexId Index ID.
+     * @param rowId Row ID.
+     */
+    public void putIndexLastBuildRowId(int partitionId, UUID indexId, @Nullable RowId rowId) {
+        try {
+            metaColumnFamily.put(indexMetaKey(partitionId, indexId), indexLastBuildRowId(rowId));
```

Comment:
Batch is encapsulated somewhere inside, right?

Reply:
Forgot about that, sorry.